### ISSUE TITLE
RudderTyper doesn't work on Node 22

### ISSUE THREAD

**Issue by princed** [rudderlabs/rudder-typer#86]
RudderTyper fails with the following error when running under Node 22, which now in [Active LTS status](https://github.com/nodejs/Release#release-schedule):

```
[...]/node_modules/rudder-typer/src/cli/index.js:18
import packageJson from '../../package.json' assert { type: 'json' };
                                             ^^^^^^

SyntaxError: Unexpected identifier 'assert'
    at compileSourceTextModule (node:internal/modules/esm/utils:340:16)
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:102:18)
    at #translate (node:internal/modules/esm/loader:433:12)
    at ModuleLoader.loadAndTranslate (node:internal/modules/esm/loader:480:27)
    at async ModuleJob._link (node:internal/modules/esm/module_job:112:19)

Node.js v22.11.0

Process finished with exit code 1
```

That is because `assert` support [has been removed from Node](https://github.com/nodejs/node/pull/52104) in favour of `with` that is based on a more recent version of [Import Attributes](https://github.com/tc39/proposal-import-attributes#history) proposal.

If I manually change all the `assert` keywords to `with`, everything works as expected.

**Comment by akashrpo** [comment]
Hi @princed, thanks for the feedback! We'll update this to ensure compatibility with Node.js v22.

**Comment by akashrpo** [comment]
Which command was used to run the generator? Can you please share more info on the steps?

**Comment by princed** [comment]
Basically any command, even `--help` or `--version`

**Node 20**
```
⋊> node --version
v20.17.0
⋊> rudder-typer --version
(node:18480) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
Version: 1.3.0
```

**Node 22**
```
⋊> node --version
v22.11.0
⋊> rudder-typer --version
[...]/node_modules/rudder-typer/src/cli/index.js:18
import packageJson from '../../package.json' assert { type: 'json' };
                                             ^^^^^^

SyntaxError: Unexpected identifier 'assert'
    at compileSourceTextModule (node:internal/modules/esm/utils:340:16)
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:102:18)
    at #translate (node:internal/modules/esm/loader:433:12)
    at ModuleLoader.loadAndTranslate (node:internal/modules/esm/loader:480:27)
    at async ModuleJob._link (node:internal/modules/esm/module_job:112:19)

Node.js v22.11.0
```

**Comment by akashrpo** [comment]
This issue has been resolved in #95. Closing this now.
